utk_shiftlog/server.lua

```diff
@@ -89,9 +99,20 @@ AddEventHandler("utk_sl:jobchanged", function(old, new, method)
 end)
 
 AddEventHandler("playerDropped", function(reason)
+    local xPlayer = ESX.GetPlayerFromId(source)

```

Comment:
this is not a proper way, when player exits the game, ESX object of that player is removed, this would only work sometimes depending on how fast the table is deleted, proper way to do it is getting the player's identifier with cfx natives.